tests/sentry/utils/performance_issues/test_consecutive_db_detector.py

```diff
@@ -252,3 +253,24 @@ def test_fingerprint_of_autogroups_match(self):
         fingerprint_2 = self.find_problems(event_2)[0].fingerprint
 
         assert fingerprint_1 == fingerprint_2
+
+    def test_respects_project_option(self):
+        project = self.create_project()
+        event = get_event("n-plus-one-api-calls/n-plus-one-api-calls-in-issue-stream")
```

Comment:
This isn't wrong per se, but should probably be a consecutive span fixture 😅

Reply:
Ohh shoot! Lol, I did copy it from your example, it was probably overlooked because I didn't think the event would be relevant for this test.
Good catch!